nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestColdStorage.java

```diff
@@ -90,23 +97,34 @@
 
     @Test
     public void shouldMoveToColdStorage() throws IOException {
-        DocumentModel documentModel = createFileDocument(DEFAULT_DOC_NAME, true);
+        // with regular user with "WriteColdStorage" permission
+        ACE[] aces = { new ACE("john", SecurityConstants.READ, true), //
+                new ACE("john", SecurityConstants.WRITE, true), //
+                new ACE("john", SecurityConstants.WRITE_COLD_STORAGE, true) };
+        DocumentModel documentModel = createFileDocument(DEFAULT_DOC_NAME, true, aces);
+
+        try (CloseableCoreSession userSession = CoreInstance.openCoreSession(documentModel.getRepositoryName(),
+                "john")) {
+            moveAndVerifyContent(userSession, documentModel);
+        }
 
+        documentModel = createFileDocument(DEFAULT_DOC_NAME, true, aces);
         // move the blob to cold storage
-        documentModel = ColdStorageHelper.moveContentToColdStorage(session, documentModel.getRef());
-        session.saveDocument(documentModel);
-        transactionalFeature.nextTransaction();
-        documentModel.refresh();
-
-        assertTrue(documentModel.hasFacet(FacetNames.COLD_STORAGE));
-
-        assertNull(documentModel.getPropertyValue(ColdStorageHelper.FILE_CONTENT_PROPERTY));
+        moveAndVerifyContent(session, documentModel);
+    }
 
-        // check if the `coldstorage:coldContent` property contains the original file content
-        Blob content = (Blob) documentModel.getPropertyValue(ColdStorageHelper.COLD_STORAGE_CONTENT_PROPERTY);
-        assertNotNull(content);
-        assertEquals(FILE_CONTENT, content.getString());
-        assertEquals("dummy", ((ManagedBlob) content).getProviderId());
+    @Test
+    public void shouldFailWhenUserDoNotHaveWriteColdStoragePermission() {
```

Comment:
For consistency, since you've renamed the test in [MoveToColdStorageTest#shouldFailWithoutRightPermissions](https://github.com/nuxeo/nuxeo/pull/3843/files#diff-cbed78a98d097ba042904ae895850590R52), don't you want to have the same name here?

Reply:
Nice catch, thanks!
Yes, clearly a side of effect of maintaining duplicated tests... 😞